### ISSUE TITLE
[ 2374975 ] New feature: log file merger

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-qconnect-base#61]
The connection configuration contains a parameter "`logfile`". This file contains the incoming messages only. Whereas the Robot Framework debug log file only contains the outgoing messages.

This makes it difficult to debug the communication. We should think about an additional application that merges the logfiles (lines merged and sorted by timestamp).

**Comment by ngoan1608** [comment]
Hi Holger,

I understand the idea — having a merged view of incoming and outgoing messages would help in debugging.
However, the logfile in qconnect-base is meant to capture only connection-level (incoming) data, keeping it separate from the test-level Robot log.

I think it’s better handled by a separate post-processing tool rather than being built into the package.
This approach keeps the framework clean and allows flexibility to develop or customize the log merger independently when needed.

Let's gather our team member's ideas before deciding how to proceed.
 
Thank you,
Ngoan

**Comment by HolQue** [comment]
Yes, of course, the logfile merger must be a separate application.

**Comment by test-fullautomation** [comment]
For me for the moment for the backlog
